generators/linting-ts/templates/test/tslint.json

```diff
@@ -0,0 +1,6 @@
+{
+  "extends": "@springworks/tslint-config",
+  "rules": {
+    "no-unused-expression": false
```

Comment:
This rule should not go in here. If it's supposed to be included, it should be in the tslint-config.

Reply:
Correct, will fix.